### DISCUSSION TITLE
How to interactively vary animationSpeed in trips example

### DISCUSSION THREAD

**Discussion by kodonnell** [visgl/deck.gl#7752] [Q&A]
See https://github.com/visgl/deck.gl/issues/7685#issue-1596097896 - I know how to manually set animationSpeed but I'd like to vary it interactively as you can with `trailLength`. I feel like this is simple to someone who knows react - any ideas?

NB: I believe it'd then be trivial to slide forward/backward in time, which would be cool.

**Comment by Pessimistress** [comment]
There are a few of examples with a playback control, e.g. https://deck.gl/examples/globe-view.
